apps/src/templates/progress/progressStyles.js

```diff
@@ -64,6 +77,216 @@ export const hoverStyle = {
     'background-color .2s ease-out, border-color .2s ease-out, color .2s ease-out'
 };
 
+/**
+ * ======================================
+ * Bubble styles
+ * ======================================
+ */
+
+export const BubbleSize = makeEnum('dot', 'letter', 'full');
```

Comment:
Mostly curious, why is 'dot' a size and not a shape?

Reply:
dots can me either circles or diamonds, and specifically refers to the size (smallest size, too small for content). i can document this enum in a subsequent PR as well.

Reply:
Oh, I thought dots was referring to the small circle.  Maybe we can call it 'tiny' or something else that conveys size but not shape?